cpl.go

```diff
@@ -0,0 +1,35 @@
+package gdal
+
+/*
+#include "go_gdal.h"
+#include "gdal_version.h"
+
+#cgo linux  CFLAGS: -I/usr/include/gdal
+#cgo linux  LDFLAGS: -lgdal
+#cgo darwin pkg-config: gdal
+#cgo windows LDFLAGS: -Lc:/gdal/release-1600-x64/lib -lgdal_i
+#cgo windows CFLAGS: -IC:/gdal/release-1600-x64/include
+*/
+import "C"
+import (
+	"unsafe"
+)
+
+// Get a configuration option.
+func GetConfigOption(key, def string) string {
+	k := C.CString(key)
+	d := C.CString(def)
+	defer C.free(unsafe.Pointer(k))
+	defer C.free(unsafe.Pointer(d))
+	opt := C.CPLGetConfigOption(k, d)
```

Comment:
Just wondering what does `CPL` mean in this context?

Reply:
CPL == [Common Portability Library](https://trac.osgeo.org/gdal/wiki/FAQGeneral#WhatdoesCPLstandfor).

BTW, this function corresponds to https://www.gdal.org/cpl__conv_8h.html#a21a509890bf6632f1b1d836c1d3bb970 in C land.